tests/cross/src/test/scala/tests/pc/SignatureHelpPatternSuite.scala

```diff
@@ -95,4 +109,24 @@ class SignatureHelpPatternSuite extends BaseSignatureHelpSuite {
        |        ^^^^^^
        |""".stripMargin
   )
+
+  check(
+    "using-params".tag(IgnoreScala2),
+    """
+      |class HKT[C[_], T](a: C[T])
+      |object HKT {
+      |  def unapply(using String)(using Boolean)(a: Int): Option[(Int, Int)] = Some(2 -> 2)
+      |}
+      |object Main {
+      |  given b: Boolean = true
+      |  given str: String = ""
+      |  (null: Any) match {
+      |    case HKT(1, @@) =>
+      |  }
+      |}
+      |""".stripMargin,
+    """|unapply(implicit x$1: String)(implicit x$2: Boolean)(a: Int): Option[(Int, Int)]
```

Comment:
For scala 3 shouldn't `implicit` be called `using`?

Reply:
can be fixed with https://github.com/scalameta/metals/pull/3707 :)